### DISCUSSION TITLE
Allow Forem Admins to remove podcasts from `/admin/content_manager/podcasts`

### DISCUSSION THREAD

**Discussion by ellativity** [forem/forem#16702] [Feature Requests]
Currently, once someone submits a podcast, it can only be removed from the admin list via the console. 

Similarly, individual episodes can't be removed from a published podcast.

Admins (and podcast creators) would like more control over the appearance of podcasts on both admin- and member-facing sides.

**Comment by michael-tharrington** [comment]
I just wrote up a related issue here - https://github.com/forem/forem/issues/18041
